models/movie.model.js

```diff
@@ -0,0 +1,127 @@
+const filename =
+  process.env.HOME + "/Desktop/Programming/Ontotext/MoviesAPI/data/movies.json";
```

Comment:
Do I have such file on my machine?

Reply:
This is the file that is in data, but in order to write to it it is required to specify the full path.

Reply:
You can get the current working directory and avoid hardcodding it. Search for CWD